shell.client/src/main/java/org/metaborg/spoofax/shell/client/IEditor.java

```diff
@@ -36,4 +36,20 @@
      */
     List<String> history();
 
+    /**
+     * Set the prompt to display.
+     *
+     * @param promptString
+     *            The prompt string.
+     */
+	void setPrompt(String promptString);
+
+    /**
+     * Set the prompt to display when in multi-line mode.
+     *
+     * @param promptString
+     *            The prompt string.
+     */
+	void setContinuationPrompt(String promptString);
+
```

Comment:
I moved them there since the Repl was attempting to change the prompt, and my changes made that impossible.
My thoughts were that setting a prompt would be universal across editors, but we don't know indeed.